src/v2/Components/Artwork/FillwidthItem.tsx

```diff
@@ -130,11 +107,16 @@ export class FillwidthItemContainer extends React.Component<
       return null
     }
 
+    const scaledImage = resized(artwork.image.url, {
+      resizeTo: artwork.image.aspectRatio === 1 ? "fill" : "fit",
```

Comment:
Ah so: there's a separate function for crop in the same file there. It'd be clearer to just leave those alone and swap out the functions I think.